src/IO/CompressionMethod.cpp

```diff
@@ -52,27 +59,28 @@ CompressionMethod chooseCompressionMethod(const std::string & path, const std::s
         return CompressionMethod::Zlib;
     if (*method_str == "brotli" || *method_str == "br")
         return CompressionMethod::Brotli;
+    if (*method_str == "LZMA" || *method_str == "xz")
```

Comment:
A small note... `xz` is actually LZMA2. But why do we need this name? Maybe just leave `xz` only.

Reply:
Is there the only place to remove LZMA?
Is it ok to leave all buffer filenames with LZMA* prefix?